src/middlewares/analyticsMiddleware/newWalletEvents.js

```diff
@@ -0,0 +1,51 @@
+import { gaSendEvent } from 'utils/analytics'
```

Comment:
I guess, it's `walletCreateEvents` name is better.

Reply:
There are both create and import events here